frontend/packages/rhoas-plugin/src/topology/components/TopologyKafkaPanel.tsx

```diff
@@ -49,12 +56,25 @@ export const ConnectedTopologyRhoasPanel: React.FC<TopologyRhoasPanelProps> = ({
     setShowAlert(false);
   };
 
+  const kindRef = referenceFor(akc);
+  const kindObj = modelFor(kindRef);
+
   return (
     <div className="overview__sidebar-pane resource-overview">
       <div className="overview__sidebar-pane-head resource-overview__heading">
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name co-resource-item">
-            <h3>{t('rhoas-plugin~Kafka Connection')}</h3>
```

Comment:
This key is not use anymore. Let's remove it
https://github.com/openshift/console/blob/master/frontend/packages/rhoas-plugin/locales/en/rhoas-plugin.json#L53

Reply:
Edit: That was the fastest review resolution in my life :)